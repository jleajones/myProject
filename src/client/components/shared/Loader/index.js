import React from 'react';

import lang from './language';

const Loader = () => <div>{lang.loading()}</div>;

export default Loader;
