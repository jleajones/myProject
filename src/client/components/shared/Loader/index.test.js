import React from 'react';
import { shallow } from 'enzyme';
import Loader from '.';

describe('<Loader />', () => {
  test('Renders component without crashing', () => {
    const component = shallow(<Loader />);

    expect(component).toMatchSnapshot();
  });
});
