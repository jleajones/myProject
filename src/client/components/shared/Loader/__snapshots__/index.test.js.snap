// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> Renders component without crashing 1`] = `
<div>
  loading
</div>
`;
